lib/Wikia/tests/Util/SamplerProxyTest.php

```diff
@@ -284,6 +288,18 @@ function testExceptionFallbackFromShadow() {
 		$this->assertEquals( $originalTestResult, $result );
 	}
 
+	function getRandomInt() {
```

Comment:
Minor: I'd add a comment or rename this method to show it's not real random. What do you think about `getFakeRandomInt` or something that mentions perfect distribution?